Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyName>Frends.Excel.ConvertToCSV</AssemblyName>
     <RootNamespace>Frends.Excel.ConvertToCSV</RootNamespace>
 
-    <Version>1.0.0</Version>
+    <Version>1.0.1</Version>
     <Authors>Frends</Authors>
     <Copyright>Frends</Copyright>
     <Company>Frends</Company>

```

Comment:
Please add a setting to include the FrendsTaskMetadata.json in the nuget package.